src/ad-sizes.test.ts

```diff
@@ -0,0 +1,20 @@
+import { _ } from './ad-sizes';
+
+const { getAdSize } = _;
+
+describe('ad sizes', () => {
+	it.each([
+		[getAdSize(2, 2), 2, 2, '2,2'],
+		[getAdSize(100, 100), 100, 100, '100,100'],
+		[getAdSize(320, 50), 320, 50, '320,50'],
+		[getAdSize(970, 250), 970, 250, '970,250'],
+		[getAdSize(0, 0), 0, 0, 'fluid'],
+	])(
+		'getAdSizes outputs correct values',
```

Comment:
You can call `getAdSize` inside each call of the test, and use %s for each run to have relevant info.

For example: `getAdSize outputs %s for %n and %n`.

Reply:
Nice, didn't know you could do that!